vnext/template/cs-app/src/App.xaml.cs

```diff
@@ -50,7 +50,12 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
         /// <summary>
         /// Invoked when the application is activated by some means other than normal launching.
         /// </summary>
+{{#useWinUI3}}
+        protected override void OnActivated(Windows.ApplicationModel.Activation.IActivatedEventArgs e)
```

Comment:
you can remove the template conditional and just always use the full type name with the namespace

Reply:
I thought that would be unneceserily ugly for the common path for now. We can reconsider if WinUI3 becomes the standard.

Reply:
Especially since the namepace is imported in non-winui3 which can give some customers warnings about unnecesary prefix depending on the style guidelines they configured.